.github/workflows/go.yml

```
@@ -13,25 +13,25 @@ jobs:
     runs-on: ubuntu-latest
     steps:
 
-    - name: Set up Go 1.x
-      uses: actions/setup-go@v2
-      with:
-        go-version: ^1.16
-
-    - name: Setup Node.js environment
-      uses: actions/setup-node@v2.1.2
-
-    - name: Check out code into the Go module directory
-      uses: actions/checkout@v2
-    
-    - name: Setup Task
-      run: curl -sL https://taskfile.dev/install.sh | sh
-    
-    - name: Build using Taskfile.yml
-      run: ./bin/task build
-    
-    - name: Archive production artifacts
-      uses: actions/upload-artifact@v2
-      with:
-        name: compiled-application-amd64-ubuntu
-        path: out/*
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v2
```

The `name` is not required, I don't recommend to name standard actions because this uses the default name `run actions/setup-go@v2` and it's easy to read for new collaborators.
```suggestion
      - uses: actions/setup-go@v2
```
I recommend name custom action like run custom scripts, or more than 1 command in the same step.

---

That's a good idea. I removed names from several steps. 👍